app/src/main/kotlin/com/softteco/template/ui/AppContent.kt

```diff
@@ -33,12 +27,6 @@ fun AppContent(
     val snackbarHostState = remember { SnackbarHostState() }
     val coroutineScope = rememberCoroutineScope()
 
```

Comment:
We will use it to send notifications to a specific user, for example. But right now we are using the deprecated way of getting an instance of FirebaseMessaging (https://firebase.google.com/docs/android/kotlin-migration#ktx-apis-to-main-how-to-migrate). And using this API leads to the described exception. The correct way is to call from packages:
```
import com.google.firebase.Firebase
import com.google.firebase.messaging.messaging
```
Also, this code only does the token logging, which we already have in AppFirebaseMessagingService.onNewToken(). It is also better to use this callback to register the token on the server, rather than fetching it from the FM instance on demand